docker-compose.yml

```diff
@@ -35,6 +35,9 @@ services:
       AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
       HEADER_ACCEPT: ${HEADER_ACCEPT}
       HEADER_X_API_KEY: ${HEADER_X_API_KEY}
+      GOOGLE_APPLICATION_CREDENTIALS: '/google/gcp_service_account.json'
+    volumes:
+      - ${GOOGLE_APPLICATION_CREDENTIALS}:/google/gcp_service_account.json
```

Comment:
In my experience adding a volume mount that uses an environment variable requires the developer to put something for the environment variable even if they don't intend on using the functionality related to the volume mount. A note should be made in the sample.env or README that the developer may run into problems spinning up containers if GOOGLE_APPLICATION_CREDENTIALS is left blank.